packages/react-components/src/helpers/validations.js

```diff
@@ -61,17 +52,7 @@ function isValidRequiredField(validator, field) {
  * Date Validations *
  */
 function isValidYear(value) {
-  return (
-    Number(value) >= 1900 && Number(value) <= moment().add(100, 'year').year()
-  );
-}
-
-function isValidYearOrBlank(value) {
-  return isValidYear(value) || value === '';
-}
-
-function isValidCurrentOrPastYear(value) {
-  return Number(value) >= 1900 && Number(value) < moment().year() + 1;
+  return Number(value) >= minYear && Number(value) <= maxYear;
```

Comment:
Here I replaced `moment` with the date const from above.

Reply:
nice use of predefined variables here!